lib/build/node-sass.js

```diff
@@ -37,12 +62,13 @@ function encodeRelativeURL(startPath, uri) {
   if (absoluteStart) {
 
     // find path to the root, stopping at cwd, package.json or bower.json
-    var pathToRoot = [ ];
-    do {
+    var pathToRoot = [];
+    var isWorking = true;
+    while (isWorking) {
```

Comment:
Keep the do-while syntax to preserve intent, just move the `isWorking` declaration outside without assignment